lib/nhsuk-prototype-kit.js

```diff
@@ -30,12 +31,35 @@ NHSPrototypeKit.middleware = middleware
 // This does all setup for both the Express app
 // and the Nunjucks environment
 NHSPrototypeKit.init = function (options) {
+  options.express = express()
+
   // Set all the Express settings
   expressSettings.setAll(options.express)
 
+  let nunjucksViews = []
+
+  if (options.appViews) {
+    nunjucksViews.push(options.appViews)
```

Comment:
```suggestion
    nunjucksViews.push(...options.appViews)
```

Reply:
`options.appViews` is currently expected to be a string (a single path) - are you thinking it could be an array? Generally I think most prototypes will only have a single `views` folder, but I guess there could be multiple?

Reply:
Ahh it's got a plural name like some of the other array options

This is why I like JSDoc validation. Should it handle both?

Reply:
Done in 46f0fa8c1e711d27fae068846f98a9d5195ddefb